src/main/webapp/partials/instructorHelpStudents.jsp

```diff
@@ -179,6 +179,9 @@
               Click <button class="btn btn-primary btn-s">Save Changes</button> to save your changes to Student A's profile.
```

Comment:
Let's not call it student's profile
![image](https://user-images.githubusercontent.com/13977235/39674145-1a3ef04c-517a-11e8-80ba-a3f70ffc7587.png)

Student profile is something student can change (and instructors cannot change but they can view). Let's call it `student details` or something to avoid confusion.